test/test_parabolic_1d.jl

```diff
@@ -87,6 +87,20 @@ end
     end
 end
 
+@trixi_testset "TreeMesh1D: elixir_advection_diffusion_cfl.jl" begin
+    @test_trixi_include(joinpath(examples_dir(), "tree_1d_dgsem",
```

Comment:
```suggestion
    @test_trixi_include(joinpath(EXAMPLES_DIR",
```
In #2493, we changed that to always use `EXAMPLES_DIR`.